plonky2/Cargo.toml

```diff
@@ -34,6 +34,7 @@ serde = { version = "1.0", default-features = false, features = ["derive", "rc"]
 serde_json = "1.0"
 static_assertions = { version = "1.1.0", default-features = false }
 unroll = { version = "0.1.5", default-features = false }
+instant = "0.1"
```

Comment:
I wonder if this breaks `no_std` builds? If so, could make it an optional dependency, with `timing` depending on `dep:instant`